### DISCUSSION TITLE
distribute the final delphivcl project issue

### DISCUSSION THREAD

**Discussion by alas82** [Embarcadero/DelphiVCL4Python#42] [Q&A]
hello , 
i follow all the steps mentioned in your tutorials , and export a form from delphi ide to python project then i tried to make the python as distributed exe using pyinstaller , the exe and dependencies as per attachment
![Capture](https://user-images.githubusercontent.com/65008114/209223088-56270ac4-4708-450c-97d6-a7b36e29cc4a.PNG)

but when you run the exe it doesnt run , please advise what is wrong

thanks

**Comment by lmbelo** [comment]
I will try it and give you guys a feedback.

**Comment by mark7p** [comment]
try using this:
pyinstaller --onefile --windowed --collect-all delphivcl .\your_python_file.py
